<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/icons/Icons/sudoku.png</normaloff>:/icons/Icons/sudoku.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>227</width>
      <height>227</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>227</width>
      <height>227</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionCheck"/>
   <addaction name="actionExit"/>
   <addaction name="actionQuestion"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/Icons/sudokuNew.png</normaloff>
     <normalon>:/icons/Icons/sudokuNew.png</normalon>:/icons/Icons/sudokuNew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Start new game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/Icons/sudokuExit.png</normaloff>:/icons/Icons/sudokuExit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close window</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/Icons/sudokuCheck.png</normaloff>:/icons/Icons/sudokuCheck.png</iconset>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
   <property name="toolTip">
    <string>Check if it is correct</string>
   </property>
  </action>
  <action name="actionQuestion">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/icons/Icons/question.png</normaloff>:/icons/Icons/question.png</iconset>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
   <property name="toolTip">
    <string>How to play?</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
